src/website/src/app/documentation/demos/dropdown/dropdown.demo.html

```diff
@@ -186,7 +186,8 @@ <h2>Accessibility</h2>
             <li>The element with the <code class="clr-code">button</code> role should also have an <code class="clr-code">aria-expanded</code>
               attribute which is set to true when the dropdown menu is open
             </li>
-            <li>The items should have the role <code class="clr-code">menuitem</code></li>
+            <li>The items should be inside an element with the role <code class="clr-code">menu</code>, and have the role <code class="clr-code">menuitem</code></li>
+            <li>If the menu uses headings, the heading should be used as an accessible description for each related item</li>
```

Comment:
Nit: Not sure if we should add a line about separators. I wouldn't like our documentation to be a comprehensive A11Y guide, but if it will be a common case it may be worth it.

Reply:
I agree with you that we need to figure out what's the right level of detail for our dox. I can add it for now with an understanding that it may be removed or expanded in a future review of dox.